bliss/utils.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import torch
+from torch import Tensor
+
+from bliss.catalog import FullCatalog
```

Comment:
Probably better to avoid creating a "utils" module, see https://www.yanglinzhao.com/posts/utils-antipattern/

Reply:
I just reverted this commit since it wasn't related to the metrics at all, I just had it sitting on the branch.